src/main/java/org/sagebionetworks/bridge/sdk/models/UploadRequest.java

```diff
@@ -89,4 +77,94 @@ public String toString() {
         return String.format("UploadRequest [name=%s, contentMd5=%s, contentType=%s, contentLength=%s]", 
                 name, contentMd5, contentType, contentLength);
     }
+
+    /** Builder for UploadRequest. */
+    public static class Builder {
```

Comment:
We should have a standard: are builders static classes of the objects they build, or separate, I think we do both across the SDK and server code.